changelogs/current.yaml

```diff
@@ -58,6 +58,10 @@ minor_behavior_changes:
   change: |
     HTTP/3 alt-svc headers will now be respected from IP-address-based hostnames. This change is
     guarded by runtime guard ``envoy.reloadable_features.allow_alt_svc_for_ips``.
+- area: http3
```

Comment:
I think you don't need to comment until the guard is true, and given it's not safe to use I'd avoid relnotes for now